scripts/launch.sh

```diff
@@ -36,6 +36,11 @@ if [[ "$TPS" != "" ]]; then
   panesCnt=$((n+1))
 fi
 
+# System start time in seconds (time since epoch).
+system_start=`date +%s`
```

Comment:
I think it should be increased by few seconds, because launching nodes takes some time. It's fine to launch nodes before system start. They will just wait for it.